voice-client-javascript/assets/quickstart.js

```diff
@@ -0,0 +1,208 @@
+$(function() {
```

Comment:
I'm sure this is just left over from the existing quickstart, but could we do away with JQuery? It's only used for this, which is just `window.addEventListener('load', ...)` and `$.getJSON` which we could replace with `fetch`.

Reply:
Yeah one step at a time...this is just a straight port at the moment.